lib/Dialect/Arc/Transforms/ConvertCombToArith.cpp

```diff
@@ -0,0 +1,241 @@
+//===- ConvertCombToArith.cpp ---------------------------------------------===//
```

Comment:
Should this be moved to the `Conversions` directory as it is a clean dialect-to-dialect conversion pass?

Reply:
Yeah great point. Let me do that.